WordPress/Classes/Networking/Remote Objects/RemoteGravatarProfile.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+open class RemoteGravatarProfile {
```

Comment:
What's the reason for having `GravatarProfile` and `RemoteGravatarProfile` as different types? Seems possible that the code consuming these won't care which specific type it is—thoughts?